assets/src/edit-story/components/canvas/karma/cloneSelection.karma.js

```diff
@@ -212,49 +211,3 @@ describe('Clone element integration', () => {
     );
   }
 });
-
-const customMatchers = {
```

Comment:
Are these just not used?

Reply:
They are used, but already defined in `karma/fixture/init.js`. So this is just duplicated code.